sdk-core/src/main/java/care/data4life/sdk/FhirService.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+package care.data4life.sdk
+
+import care.data4life.crypto.GCKey
+import care.data4life.crypto.error.CryptoException.DecryptionFailed
+import care.data4life.crypto.error.CryptoException.EncryptionFailed
+import care.data4life.fhir.Fhir
+import care.data4life.fhir.FhirParser
+import care.data4life.fhir.stu3.model.DomainResource
+import care.data4life.fhir.stu3.model.FhirElementFactory
+import care.data4life.sdk.lang.D4LException
+import io.reactivex.Single
+
+// TODO remove @JvmOverloads when RecordService and Client changed to Kotlin
+// TODO use of Single is not necessary as it's finalized with blockingGet()
+internal class FhirService @JvmOverloads constructor(
+        private val cryptoService: CryptoService,
+        private val parserFhir3: FhirParser<Any> = Fhir().createStu3Parser()
```

Comment:
`fhir3Parser` reads a bit nicer, in my opinion.

Reply:
I work on that in the FHIR 4 branch

Reply:
👍